build/DownloadDependencies/CommonProperties.props

```diff
@@ -31,6 +31,13 @@
     <BaseRuleDescFileTargetDirectory>$(MSBuildThisFileDirectory)..\..\src\Rules\Embedded\</BaseRuleDescFileTargetDirectory>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <SloopBinaryFileName>sonarlint-backend-cli-$(EmbeddedSLOOPVersion)-windows_x64.zip</SloopBinaryFileName>
+    <SloopBinaryDownladDir>$(LOCALAPPDATA)\SLVS_Build_SLOOP</SloopBinaryDownladDir>
+    <SloopBinaryDownlaoadPath>$(SloopBinaryDownladDir)\$(SloopBinaryFileName)</SloopBinaryDownlaoadPath>
+    <SloopBinaryRemoteUrl>https://repox.jfrog.io/artifactory/sonarsource/org/sonarsource/sonarlint/core/sonarlint-backend-cli/$(EmbeddedSLOOPVersion)/$(SloopBinaryFileName)</SloopBinaryRemoteUrl>
```

Comment:
SLOOP vs Sloop

Reply:
I prefer `Sloop` here. The MS guildeline is to use Xxx for abbreviations that have 3 or more characters. Like SqlConnection, or SmtpClient instead of SQLConnection and SMTPClient